src/components/userGoals/services/userGoals.service.js

```diff
@@ -1,87 +1,92 @@
 'use strict';
 
-angular.module('znk.infra-web-app.userGoals').provider('UserGoalsService', [function() {
+angular.module('znk.infra-web-app.userGoals').provider('UserGoalsService', [function () {
 
-        var _calcScoreFn;
-
-        this.setCalcScoreFn = function(calcScoreFn) {
-            _calcScoreFn = calcScoreFn;
+    //function expects userGoals object
```

Comment:
Why did you change this function?   why is it needed?
It only returns the "totalScore" ?